client/core/nd.lua

```diff
@@ -32,39 +31,48 @@ end)
 
 local shared = exports[coreName]
 
-local coreFunctionsOverride = {
-    playerData = {
-        originalMethod = 'getPlayer',
-        modifier = {
-            executeFun = true,
-            effect = function(originalFun)
-                while not loaded do
-                    Wait(1000)
-                end
-                local data = originalFun()
-                local jobData = data.jobInfo
+function Core.getPlayerData()
+    local player = shared.GetPlayerData()
+    local job = player.job
+    local gang = player.gang
+    local charinfo = player.charinfo
+    local year, month, day = charinfo.birthdate:match('(%d+)-(%d+)-(%d+)')
 
-                local year, month, day = data.dob:match("(%d+)-(%d+)-(%d+)")
+    local formattedJob = {
+        name = job.name,
+        label = job.label,
+        onDuty = job.onduty,
+        isBoss = job.isboss,
```

Comment:
The original key was `isBoss` and now this one is `isboss` . A lot of other things in ND is also not the same as the previous table structure. Can you please double check.

Reply:
isnt that how boss thing works for esx and nd bcz i forgot i stoped using it